exchanges/account/account.go

```diff
@@ -32,7 +32,8 @@ var (
 	errNoCredentialBalances         = errors.New("no balances associated with credentials")
 	errCredentialsAreNil            = errors.New("credentials are nil")
 	errOutOfSequence                = errors.New("out of sequence")
-	errUpdatedAtIsZero              = errors.New("updated at is zero")
+	errUpdatedAtIsZero              = errors.New("updatedAt may not be zero")
+	errLoadingBalance               = errors.New("cannot loading balance")
```

Comment:
🚧  Just a minor typo here:
```suggestion
	errLoadingBalance               = errors.New("cannot load balance")
```
Alternative: `error loading balance`